crates/polars-io/src/csv/write_impl.rs

```diff
@@ -23,31 +23,33 @@ use super::write::QuoteStyle;
 fn fmt_and_escape_str(f: &mut Vec<u8>, v: &str, options: &SerializeOptions) -> std::io::Result<()> {
     if options.quote_style == QuoteStyle::Never {
         write!(f, "{v}")
```

Comment:
Could you do a return here? Then the rest of the function can have 1 less indentation (remove the subsequent else-block).
```suggestion
        return write!(f, "{v}")
```

Reply:
Sure, as you like... ;)